src/collections/members/vinayak-sharma/index.mdx

```diff
@@ -6,4 +6,5 @@ twitter: Vinayak47427793
 github: vinayaksh42
 blurb: Mesh Head
 status: Active
+Linkedin: in/vinayak-sharma-141096193/
```

Comment:
Convenient that people don't need to include the full URL. If we're going to do this, let's take the final step and not require the `in/` section either. What do you think, @vinayaksh42?